roles/kokonet-setup/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Configure eth1 for data plane
+  template:
+    src: ifcfg-eth1.j2
+    dest: /etc/sysconfig/network-scripts/ifcfg-eth1
+  notify: "restart net"
```

Comment:
do we need to restart network service here? if you are just trying to bring up eth1, can we not do ifup eth1?